index.d.ts

```diff
@@ -13,6 +13,13 @@ declare namespace npmRunPath {
 		Set it to an empty string to exclude the default PATH.
 		*/
 		readonly path?: string;
+
+		/**
+		Path to the current Node.js executable. Its directory is pushed to the front of PATH.
```

Comment:
Maybe include some use-cases here? Not clear to me as a user what I would use it for. Should also mention that the path can be relative. Then it's relative to `cwd`.

Reply:
Fixed in 577f0385c1bed8acb1263d425cdf3786e112df45